Task3/download_contributors_avatars.py

```diff
@@ -0,0 +1,59 @@
+﻿"""This module creates {username}/{project} folders
+and download the avatar picture files with names
+that match GitHub contributor’s login"""
+import argparse
+import json
+import os
+import requests
+
+USER_NAME = 'ViraBakhovska'

```

Comment:
Please add two more arguments to the argparser for your username/token.

Reply:
added arguments for username/token.